knowledge-base/docs/fundamentals/apply_col_default_to_existing_row.md

```diff
@@ -0,0 +1,60 @@
+---
+id: apply_col_default_to_existing_row
+title: Apply column default to existing rows
+description:
```

Comment:
This line throws the following error during the `docusaurus build` command:

```
[ERROR] The following front matter:
---
id: apply_col_default_to_existing_row
title: Apply column default to existing rows
description: null
tags:
- beginner
- getting-started
---
contains invalid values for field(s): `description`.
```

And it prevents vercel to deploy the changes. We should include a description to fix the error.

Reply:
thanks @guerrero - yes the description fields need to be completed, I had been leaving those blank for @tb-peregrine to complete, so I could focus on getting through the rest of the content